R/otp-batch.R

```diff
@@ -0,0 +1,98 @@
+#' Get multiple routes from the OTP
```

Comment:
@mem48 I'm not sure this is the way to go with batch processing. This is likely to be a requirement of many of the functions, so it would be better if there was a single internal function which the other functions can call. The user should be able to provide either a single from/to set of coordinates OR a matrix -  in all relevant functions. Otherwise there will be a lot of repetition. Suggest moving this out of the pull request for now or at least not exporting any functions from it.